UI/data/locale/en-US.ini

```diff
@@ -827,6 +827,10 @@ Basic.Settings.Audio.MeterDecayRate.Slow="Slow (Type II PPM)"
 Basic.Settings.Audio.PeakMeterType="Peak Meter Type"
 Basic.Settings.Audio.PeakMeterType.SamplePeak="Sample Peak"
 Basic.Settings.Audio.PeakMeterType.TruePeak="True Peak (Higher CPU usage)"
+Basic.Settings.Audio.MeterUpdateRate="Meter Update Rate"
+Basic.Settings.Audio.MeterUpdateRate.TT="Controls the rate at which the volume meters are updated (the volume meters' "frame rate"). A higher rate will improve the visual response of the volume meters at the cost of a small amount of CPU usage. Default: 30 Hz"
```

Comment:
This probably requires a couple newlines (\n).

Reply:
Added newlines and fixed issue with unescaped `"` characters.